chapters/equations.tex

```diff
@@ -777,10 +777,11 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 \end{nonnormative}
 
 If a parameter has a value for the \lstinline!start!-attribute, does not have \lstinline!fixed = false!, and neither has a binding equation nor is part of a record having a binding equation, the value for the \lstinline!start!-attribute can be used to add a parameter binding equation assigning the parameter to that \lstinline!start! value.
-In this case a diagnostic message is recommended in a simulation model.
+In this case a diagnostic message is recommended in a simulation model, unless the parameter has a \lstinline!Dialog(enable)! annotation set to false.
```

Comment:
I was looking for other places where we refer to a member of an annotation like this, but couldn't find any.  To me, it would look more natural to use the non-nested form seen in proper modifications:
```suggestion
In this case a diagnostic message is recommended in a simulation model, unless the parameter has a \lstinline!Dialog.enable! annotation set to false.
```